packages/shim/shim.spec

```
@@ -17,11 +17,14 @@ Summary: UEFI shim loader
 License: BSD-3-Clause
 URL: https://github.com/rhboot/shim/
 Source0: https://github.com/rhboot/shim/archive/%{shimver}/shim-%{shimver}.tar.bz2
+Source1: https://github.com/rhboot/shim/archive/%{shimver}/shim-%{shimver}.tar.bz2.asc
```

These URLs return 404, can point them to the releases path instead?

```suggestion
Source1: https://github.com/rhboot/shim/releases/download/%{shimver}/shim-%{shimver}.tar.bz2
Source1: https://github.com/rhboot/shim/releases/download/%{shimver}/shim-%{shimver}.tar.bz2.asc
```

on that note, I assume key verification working by finding the sources, keyring, and signature files locally first? I've been checking via URLs directly in reviewing these changes.

---

> I assume key verification is working by finding the sources, keyring, and signature files locally first?

I've been doing a hybrid approach - first verifying sources using the associated signature file and my existing GPG keyring, and then exporting just the public key it references using this script:
```
#!/bin/bash
set -euo pipefail

OUTPUTDIR="${PWD}"
WORKDIR="$(mktemp -d)"
cd "${WORKDIR}"
cleanup() {
  rm -rf "${WORKDIR}"
}
trap 'cleanup' EXIT

KEY="${1:-}"
export GNUPGHOME="${WORKDIR}"
gpg --recv-keys "${KEY}"
gpg \
  --export \
  --armor "${KEY}" \
  > "${OUTPUTDIR}/gpgkey-${KEY}.asc"
```

Coincidentally, I saw the [`sq` 1.0 announcement](https://sequoia-pgp.org/blog/2024/12/16/202412-sq-1.0/) last night, which looks like a nicer way of working with some of these files.